TypeCobol.Test/Parser/Incremental/BasicEdits/AddEmptyLineAtEndThenRemove/Typedef2.tcbl

```diff
@@ -0,0 +1,18 @@
+﻿IDENTIFICATION DIVISION.
```

Comment:
Can you rather use this Cobol code and change the file name to MyPgm.rdz.cbl:
```cobol
       IDENTIFICATION DIVISION.
       PROGRAM-ID. MyPgm.












       PROCEDURE DIVISION.
       01 OutOfPerimeter2.
          02 OutOfPerimeter2-var1 PIC X.
          02 OutOfPerimeter2-var2 PIC X.

```
inc should be the same than before.

I prefer to avoid usage of TypeCobol reference if the bug is related to Cobol.